content/docs/12.5-vertex/60-system-management/10-tenant-management.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tenant Management"
+metaTitle: "Tenant Management"
+metaDescription: "Learn how to create and remove tenants in Palette VerteX."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Overview
+
+Tenants are isolated environments in Palette VerteX that contain their own clusters, users, and resources. You can create multiple tenants in Palette VerteX to support multiple teams or projects. Instructions for creating and removing tenants are provided below.
+
+
+<br />
+
+# Create a Tenant
+
+You can create a tenant in Palette VerteX by following these steps.
+
+
+## Prerequisites
+
+* Access to the Palette VerteX system console.
+
+
+## Enablement
+
+1. Log in to the Palette VerteX system console.
```

Comment:
Do you have to be a tenant admin to do this? "Log in as tenant admin ..."

Reply:
Nope, just a regular user but the first user is an admin users with tenant permissions.